src/framework/auth/services/token.service.ts

```diff
@@ -15,7 +15,7 @@ import { deepExtend, getDeepFromObject, urlBase64Decode } from '../helpers';
 @Injectable()
 export class NbAuthSimpleToken {
```

Comment:
We also would need tests for the token classes also.

Reply:
done